Automattic/wp-calypso
# Stats: Adding basic controls for the new date picker control

Related to #81936

## Proposed Changes

* adding a new component to verify the ability to navigate to a custom date

## Testing Instructions



* verify that the page on the live branch is not broken and you can navigate to a day other than the default day on the Traffic page
* apply `stats/date-control` feature flag to the live branch
* open the new dropdown and update the first input to a custom date
* click apply and verify that the chart reloads to your specified day


## Pre-merge Checklist
